src/projectStatus.ts

```diff
@@ -180,6 +192,27 @@ class TreeDataProvider implements vscode.TreeDataProvider<Node>, vscode.Disposab
         await this.refresh();
     }
 
+    public async refreshConfigNode(): Promise<any> {
+        if (this.configNode) {
```

Comment:
probably we could diminish code duplication by doing:

```ts
public async refreshNode(node: Node): Promise<any>
{ 
  if (node} { 
    await node.refresh(); 
    this.Data.Fire(node);
  }
}
```

Reply:
Good idea for sharing code, making the change.